addons/purchase_requisition/views/purchase_requisition_views.xml

```diff
@@ -2,76 +2,6 @@
-        <field name="model">purchase.requisition.type</field>
-        <field name="arch" type="xml">
-            <tree string="Purchase Agreement Types">
-                <field name="sequence" widget="handle"/>
-                <field name="name"/>
-                <field name="exclusive"/>
-            </tree>
-        </field>
-    </record>
-
-    <record id="view_purchase_requisition_type_kanban" model="ir.ui.view">
-        <field name="name">purchase.requisition.type.kanban</field>
-        <field name="model">purchase.requisition.type</field>
-        <field name="arch" type="xml">
-            <kanban class="o_kanban_mobile">
-                <field name="name"/>
-                <field name="exclusive"/>
-                <templates>
-                    <t t-name="kanban-box">
-                        <div t-attf-class="oe_kanban_global_click">
-                            <div class="o_kanban_record_top ">
-                                <div class="o_kanban_record_headings mt4">
-                                    <strong class="o_kanban_record_title"><field name="name"/></strong>
-                                </div>
-                                <field name="exclusive" widget="label_selection"/>
-                            </div>
-                        </div>
-                    </t>
-                </templates>
-            </kanban>
-        </field>
-    </record>
-    <record model="ir.ui.view" id="view_purchase_requisition_type_form">
-        <field name="name">purchase.requisition.type.form</field>
-        <field name="model">purchase.requisition.type</field>
-        <field name="arch" type="xml">
-            <form string="Purchase Agreement Types">
-            <sheet>
-                <widget name="web_ribbon" title="Archived" bg_color="text-bg-danger" invisible="active"/>
-                <group>
-                    <group string="Agreement Type">
-                        <field name="name"/>
-                        <field name="exclusive" widget="radio"/>
-                        <field name="active" invisible="1"/>
-                    </group>
-                    <group string="Data for new quotations">
-                        <field name="line_copy" widget="radio"/>
-                        <field name="quantity_copy" widget="radio"/>
-                    </group>
-                </group>
-            </sheet>
-            </form>
-        </field>
-    </record>
-    <record model="ir.ui.view" id="view_purchase_requisition_type_search">
-        <field name="name">purchase.requisition.type.search</field>
-        <field name="model">purchase.requisition.type</field>
-        <field name="arch" type="xml">
-            <search>
-                <field name="name"/>
-                <separator/>
-                <filter name="archived" string="Archived" domain="[('active', '=', False)]"/>
-            </search>
-        </field>
-    </record>
```

Comment:
@ticodoo  since i have already add a `remove_model` script so no need to add a `remove_view`.
as `remove_view` already handled in `remove_model` method  : https://github.com/odoo/upgrade-util/blob/master/src/util/models.py#L85

Reply:
Ah didn't know that! Thanks